postgres/pgdump.go

```diff
@@ -306,6 +314,15 @@ func processInsertStmt(conv *internal.Conv, n nodes.InsertStmt) *copyOrInsert {
 		logStmtError(conv, n, fmt.Errorf("can't get table name: %w", err))
 		return nil
 	}
+	if _, ok := conv.SrcSchema[table]; !ok {
```

Comment:
I don't see how this helps.

My understanding is that when we encounter a table with INHERITS, we still create the table and add it to conv.SrcSchema, but the table doesn't include any of the parent tables columns. So if we have:

  create table test_inherits () inherits (test);

then we get a table test_inherits with no columns. But we do add an entry to conv.SrcSchema, so ok will be true, which means we'll never execute the body of the if statement. I've run a few tests to confirm this (for both INSERT and COPY).

For COPY and INSERT, the insert/copy code treats the issue as a bad row and puts the rows in the dropped rows file. I think that is a reasonable approach and I don't think we need to tweak it.

I'm going to look more at the handling of foreign keys and INHERITS.